CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# _2.8_ (2024-02-20)
+- **(Bug Fix)** Amazon Q: Service exceptions are not suppressed and displayed to the user.
+
```

Comment:
again, this file is not intended to be modified directly
please use the `./gradlew :newChange --console plain` task as documented in CONTRIBUTING.md

Reply:
apologies, updated it.